R/lfd.r

```diff
@@ -141,4 +147,4 @@ sums <- dfb %>%
   summarise_at(vars(positive, negative, total), sum)
 
 fp <- qbinom(c(0.025, 0.5, 0.975), sums$total, (1 - res$estimate))
-sums$positive - fp
+sums$positive - fp
```

Comment:
Who ends a file with a line of code?

Reply:
Someone deeply traumatised by the potential for hidden white space